### PR TITLE
Resolve system variables in actions

### PR DESCRIPTION
**Pull request by manasdk**
- Support for specifying system variable stored in kv-store as default values of variables.

### CODE REVIEW

File: st2actions/st2actions/utils/param_utils.py [hunk 1]
```diff
@@ -108,14 +111,18 @@ def get_resolved_params(runnertype_parameter_info, action_parameter_info, action
...
     template = Template(template_str)
-    return template_str != template.render({})
+    try:
+        return template_str != template.render({})
+    except exceptions.UndefinedError:
+        return True
 
 
-def _renderable_context_param_split(action_parameters, runner_parameters):
+def _renderable_context_param_split(action_parameters, runner_parameters, base_context=None):
```
Comment (Kami) [inline st2actions/st2actions/utils/param_utils.py]: Can base_context contain nested dictionaries? If it can, we should use copy.deepcopy.

**Comment by Kami** [comment]
Nice.

Let's also update CHANGELOG.

**Comment by manasdk** [comment]
Oops!

Changelog updated at 78015cda8b68507e224c8aecf8518edca6eaa96e

**Comment by manasdk** [comment]
Travis breakage is unrelated to change -
Hash of the package https://pypi.python.org/packages/2.7/S/Sphinx/Sphinx-1.2.3-py2-none-any.whl#md5=7a4924447e6675d88458d41c8c93b363 (from https://pypi.python.org/simple/sphinx/) (72273d641449f9ec0ea516e11557a0c1) doesn't match the expected hash 7a4924447e6675d88458d41c8c93b363!
  Bad md5 hash for package https://pypi.python.org/packages/2.7/S/Sphinx/Sphinx-1.2.3-py2-none-any.whl#md5=7a4924447e6675d88458d41c8c93b363 (from https://pypi.python.org/simple/sphinx/)

**Comment by Kami** [comment]
LGTM, +1

**Pull request merged**